### DISCUSSION TITLE
Plugins - Attempts to add a temperature sensor for Intel Arc Graphics.

### DISCUSSION THREAD

**Discussion by juzi5201314** [Rem0o/FanControl.Releases#1593] [Show and tell]
https://github.com/juzi5201314/FanControl.fan_control_arc

**Comment by Rem0o** [comment]
Hi @juzi5201314 

Made some c# bindings from the c++ api provided by https://github.com/intel/drivers.gpu.control-library
https://github.com/Rem0o/drivers.gpu.control-library.bindings

This should be a lot more straight forward than your method with a rust layer to get a plugin going.

The bindings are automatically created using Swig.

**Reply by juzi5201314** [reply]
good, looking forward to arc support being included in the next FanControl release👍

**Reply by Rem0o** [reply]
Actually, I will first release it as a plugin, and I will need some testers, as I don't have a ARC card on hand. If the plugin is stable enough, I will include it in the main software and/or add it into LHM. That's the roadmap.

**Reply by characharm** [reply]
can't wait to test

**Reply by Rem0o** [reply]
https://github.com/Rem0o/FanControl.IntelCtlLibrary
